package.json

```diff
@@ -64,6 +64,7 @@
         "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-react": "^7.29.4",
         "husky": "^4.3.0",
+        "is-mobile": "^3.1.1",
```

Comment:
I think this becomes an actual dependency, rather than a development one?

Good job on [finding something so tiny](https://bundlephobia.com/package/is-mobile@3.1.1).

Reply:
We looked into [how `microbundle` decides whether dependencies should be external or inline](https://github.com/developit/microbundle/wiki/How-Microbundle-decides-which-dependencies-to-bundle). For this small a lib (806B minified + gzipped) it makes sense to inline it.

> ## How Microbundle decides inline _vs_ external
> 
> Microbundle decides whether a dependency should be inlined or left as an `import` / `require()` based on how you declare dependencies in your `package.json`.
> 
> When you import a module in your library, Microbundle checks to see how that module was added to your `package.json`.
> 
> - If the module is listed in the `"peerDependencies"` or `"dependencies"` fields, it will be considered _external_ and won't be inlined into your bundled code. External modules remain runtime dependencies of your bundle using `require()` or `import`.
> - Conversely, if the imported module is _only_ referenced in the `"devDependencies"` `package.json` field, that dependency module will be inlined into the bundle.
> 
> This behaviour makes sense when we think about what those fields express: when someone installs your npm package, your listed `"dependencies"` are downloaded too - the expectation is that your package will import or require those modules. The same is true for `"peerDependencies"`, which are automatically downloaded by npm version 7, and produced warnings in earlier npm versions if not already installed.

Reply:
Thanks for adding the explainer Max.